app/controllers/application_controller.rb

```diff
@@ -2,4 +2,13 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def dashboard
```

Comment:
rather than having this in the application controller, i'd suggest creating a `home_controller` or even a `dashboard_controller` to encapsulate this action.  as it stands, this method is available within every "child" controller - aka every controller you have defined since everything is inheriting from `ApplicationController`